spec/views/cases/data_requests/send_email_html_slim_spec.rb

```diff
@@ -38,7 +38,26 @@
 
       it "has required content" do
         expect(page.page_heading.heading.text).to eq "Are you sure you want to send the commissioning email?"
-        expect(page.page_banner.text).to include "The selected location does not have an email address."
+        expect(page.page_banner.text).to include "The selected location does not have an email address. Please update or select another."
+        expect(page.button_send_email.disabled?).to eq true
+        expect(page.link_cancel.text).to eq "Cancel"
+      end
+    end
+
+    context "with data request contact with only the branston probation records email given" do
```

Comment:
Good to see a test! As we've already tested for other parts of the page being present, I think it would be find for this test to just test that the error banner is present and that the button is disabled.